### PR TITLE
Rename person providers

### LINKED ISSUE #62: Switch to English translations from localised class/ field names
Some classes, such as `PeselProvider`, contain names of number types in national languages. This is not really readable for anyone that does not know a given language and imo should be switched to an English name preceded by a country code prefix.

### PR DESCRIPTION
**Pull request by andidev**
@OlgaMaciaszek
the rest of the translation work that closes #62

### CODE REVIEW

**Comment by OlgaMaciaszek** [comment]
Thanks @andidev . The changes look good, but I think you could add comments, with information that the number represents SocialSecurityNumber (for EN) or (DNI) for ES. I would avoid excessive comments in general, but in case of such translated concepts maybe it will be useful.

File: src/main/java/io/codearte/jfairy/producer/person/locale/es/EsNationalIdentityCardNumberProvider.java [hunk 1]
```diff
@@ -6,16 +6,18 @@
 import java.util.regex.Pattern;
 
 /**
+ * Spanish National Identity Card Number (known as Documento Nacional de Identidad or DNI)
```
Comment (andidev) [inline src/main/java/io/codearte/jfairy/producer/person/locale/es/EsNationalIdentityCardNumberProvider.java]: @OlgaMaciaszek 
DNI is commented here. Is this not enuogh? Not sure where else you ment it should be commented.

File: src/main/java/io/codearte/jfairy/producer/person/locale/en/EnNationalIdentityCardNumberProvider.java [hunk 2]
```diff
@@ -13,10 +13,12 @@
 import static org.apache.commons.lang3.StringUtils.leftPad;
 
 /**
+ * English National Identity Card Number (known as Social Security Number)
```
Comment (andidev) [inline src/main/java/io/codearte/jfairy/producer/person/locale/en/EnNationalIdentityCardNumberProvider.java]: @OlgaMaciaszek 
The same with Social Security Number which is commented here.

**Comment by OlgaMaciaszek** [comment]
LGTM.

**Pull request merged**